src/theory/quantifiers/instantiate.h

```diff
@@ -146,6 +146,29 @@ class Instantiate : public QuantifiersUtil
                         bool mkRep = false,
                         bool modEq = false,
                         bool doVts = false);
+  /**
+   * Same as above, but we also compute a vector failMask which contains
+   * information about what values in terms led to the instantiation not being
```

Comment:
"contains information" kinda indicates that it's more than binary. Maybe say something like "failMask marks which values in terms *did not* contribute to the instantiation ..."? The important information is what terms were irrelevant, as by default we say all were.